src/style.css

```diff
@@ -0,0 +1,67 @@
+input {
+  width: 100%;
+  font-family: Avenir, sans-serif;
+}
+
+.icon {
+  background: url(images/Search.svg);
+  background-repeat: no-repeat;
+}
+
+.icon-big {
+  background-position: 26px 25px;
+  background-size: 19px 19px;
+}
+
+.icon-small {
+  background-position: 13px 15px;
+  background-size: 11px 11px;
+}
+
+.search-big {
+  height: 70px;
+  margin-top: 71px;
+  margin-left: 48px;
+  margin-right: 81px;
+  font-size: 16px;
+  line-height: 22px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  box-sizing: border-box;
+  padding: 25px 709px 23px 62px;
+}
+
+.search-big:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search-big:focus {
+  background: linear-gradient(180deg, #fff 0%, #e6e6e7 100%);
+  border-radius: 3px;
+  font-weight: bold;
+}
+
+.search-small {
+  height: 42px;
+  margin-left: 48px;
+  margin-top: 40px;
+  padding: 12px 226px 10px 33px;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
```

Comment:
We don't have any basic search bar, so why we should use there big and small search class names? We have search-input and search-input - small (--small). Therefore, we set all styles that are supposed to be for the "large" input for all inputs in general, and with the modifier small we change only the necessary values to make our input smaller.